src/olympia/blocklist/tests/test_cron.py

```diff
@@ -218,6 +267,19 @@ def test_force_base_option(self, should_reset_mock):
         # no stash because we're starting with a new base mlbf
         assert not os.path.exists(os.path.join(gen_path, 'stash.json'))
 
+        self.statsd_incr_mock.assert_has_calls([
+            mock.call(f'{STATSD_PREFIX}blocked_changed', 0),
+            mock.call(f'{STATSD_PREFIX}success'),
+            # 2nd execution
+            mock.call(f'{STATSD_PREFIX}blocked_changed', 0),
```

Comment:
Will this actually check that this was called twice, or will this statement also return true if it was only called once? I.e., could the same call to the mock cause line 271 and line 274 to return true?

Reply:
yeah, `assert_has_calls` checks (by default, anyway) for exactly these calls, in exactly this order.